modules/ROOT/pages/index.adoc

```diff
@@ -4,7 +4,13 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-API Community Manager (ACM) overview here.
+MuleSoft enables your company to build APIs, catalog them with Exchange, and present them to software developers inside and outside your company with API Community Manager (ACM).
+
+ACM relies on MuleSoft's Anypoint Exchange as the single source of truth about published APIs, and Salesforce Community Cloud's ability to add customization, branding, and marketing information and to serve different content to different audiences.
```

Comment:
...Salesforce Community Cloud's ability to add customization, branding, and marketing information to serve unique content to different audiences.

(suggestion only)

Reply:
Updated to that version.